client/src/pages/policy/[policyId].tsx

```diff
@@ -0,0 +1,3 @@
+import { PolicyPage } from '../../modules/policy';
+
+export default PolicyPage;
```

Comment:
This is regarding the file itself, not the code. Is there need for a dynamic route `/policy/{whatever}`?